### ISSUE TITLE
genome-wide smrna detection and quantification

### ISSUE THREAD

**Issue by lpantano** [nf-core/smrnaseq#409]
### Description of feature

There are many other kind of smRNA in smRNAseq data besides miRNA, some people will trough them away, but some other will use it to investigate new hot-spots in the genomes. bcbio-nextgen has been widely used to characterize smrnas. There is one sub-workflow that would be nice to add here:

- use of seqcluster to detect hot-spots over the genome: this tool will get an unified BAM file to detect clusters over the genome an annotate them.

We are already using seqcluster, it is a local module though. We could implement this adding secluster to nf-core module.

I would like to know, @apeltzer, @atrigila , @nschcolnicov, if there is any consideration besides the nf-core module to have before working on this.

It would be an extra process, not sure if it is worth a sub-workflow.

Thanks

**Comment by apeltzer** [comment]
Yes let's add this too. Sounds like something also for the next release
